src/ui/public/vislib/visualizations/gauges/simple.js

```diff
@@ -218,7 +218,8 @@ export function SimpleGaugeProvider() {
             return `${percentage}%`;
           }
           if (d.aggConfig) {
```

Comment:
If statement needs to be updated as well. Its checking for d.aggConfig but the internals are using d.aggConfigResult.aggConfig